boards/wio_terminal/Cargo.toml

```diff
@@ -49,6 +53,7 @@ default = ["atsamd-hal/samd51p", "rt", "unproven", "usb"]
 rt = ["atsamd-hal/samd51p-rt", "cortex-m-rt"]
 unproven = ["atsamd-hal/unproven"]
 usb = ["atsamd-hal/usb", "usb-device", "usbd-serial"]
+sdcard = ["embedded-sdmmc"]
```

Comment:
Is there any reason `sdcard` shouldn't be a default feature?

Reply:
No strong inclination - Ive removed that feature as its not really a biggie for it to be a dep.